sapl/base/views.py

```diff
@@ -395,8 +395,23 @@ def get_context_data(self, **kwargs):
 
         context['title'] = _('Matérias em Tramitação')
 
+        qr = self.request.GET.copy()
         qs = context['object_list']
         qs = qs.filter(em_tramitacao=True)
+
+        if 'tramitacao__unidade_tramitacao_destino' in qr and qr['tramitacao__unidade_tramitacao_destino'] != '' :
+            id_materias = []
+            for item in qs:
+                if str(item.tramitacao_set.order_by('-id').first().unidade_tramitacao_destino_id) == qr['tramitacao__unidade_tramitacao_destino']:
+                    id_materias.append(item.id)
+            qs = qs.filter(em_tramitacao=True, id__in=id_materias)
```

Comment:
Tem certeza que isso funciona? Ao fazer `for item in qs` o QuerySet vai ser materializado, isto é, retornará uma lista então como vai chamar `filter` nela?

Reply:
Funciona. Entendo que trata o item como um registro do qs, e é necessário para poder comparar com a tramitação atual da matéria. 
Pode ocorrer até três chamadas do filter, mas qs permanece como QuerySet.